src/cow-react/modules/swap/helpers/getEthFlowEnabled.ts

```diff
@@ -0,0 +1,5 @@
+const ETH_FLOW_ENABLED_LOCAL_STORAGE_KEY = 'enableEthFlow'
+
+export function getEthFlowEnabled(): boolean {
+  return localStorage.getItem(ETH_FLOW_ENABLED_LOCAL_STORAGE_KEY) === '1'
```

Comment:
Right now a simple function. Might need to turn it into a hook for the next task when considering smart contract wallets.
But that's a different task, don't want to prematurely complicate